--- a/gcc/config.gcc	2018-01-20 08:19:02.188686071 -1000
+++ b/gcc/config.gcc	2018-01-20 09:12:01.258664249 -1000
@@ -1056,6 +1056,13 @@
 	    tmake_file="$tmake_file arm/t-linux-androideabi"
 	    ;;
 	esac
+	case ${target} in
+	arm*-*-*eabihf)
+	    with_cpu=${with_cpu:-cortex-a7}
+	    with_fpu=${with_fpu:-neon-vfpv4}
+	    with_float=${with_float:-hard}
+	    ;;
+	esac
 	# The EABI requires the use of __cxa_atexit.
 	default_use_cxa_atexit=yes
 	with_tls=${with_tls:-gnu}
